packages/docs/docs/api/fhir/operations/resend.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 7
```

Comment:
consider adding subscription tag to the header.  

Also link from /docs/subscriptions may be useful